### DISCUSSION TITLE
rAudio not adding tracks to playlist

### DISCUSSION THREAD

**Discussion by Lubos-V** [rern/rAudio#1706] [Bug Reports]
Hi there! 

Since a couple of weeks I am observing a peculiar behaviour. My rAudio installation (kept up to date) is available on the network, connected via a cable to a network bridge (together with my NAS) - I can always browse through it without any problems. After several hours of "inactivity" (i.e., coming to it in the morning in order to start listening to music), adding any playable "object" to the playing queue shows the "popup" (track / album / internet raudio added to queue) but switching to the playlist doesn't show anything there and there of course isn't any playback.

It seems that the system is trying to do something in the background as after some time (a couple of minutes?) things start working again (i.e., tracks can be added, but the previous activity is lost).

There was an update available today - it was obvious that after pressing the "update" button it took significantly longer for the system to start loading the update files (10-20 sec.) than usual.

Any idea why this might be happening, considering the fact that the interface is perfectly responsive?

**Comment by Lubos-V** [comment]
hm - interestingly enough: current playback just ended. i tried adding additional tracks to the list. the same behavior. the UI seemed to reload a minute (or so) later. after that, adding new things to playlist (and playback) works again...

**Comment by Lubos-V** [comment]
anyone else having this problem? @rern any thoughts from your side what might be causing this? it feels like the device can be pinged from outside but needs time to (re)-establish outgoing connection properly (even though it's connected to the network permanently)...

**Reply by RamblerKevin** [reply]
I might be having a similar issue, just started a new thread with my experience.

**Reply by Lubos-V** [reply]
@RamblerKevin for me it works again since then but no idea why - haven't changed anything. good luck with resolving your issue. it might be helpful referencing / linking the issues...
